pkg/loop/config.go

```diff
@@ -11,13 +11,20 @@ import (
 )
 
 const (
-	envDatabaseURL            = "CL_DATABASE_URL"
-	envPromPort               = "CL_PROMETHEUS_PORT"
+	envDatabaseURL = "CL_DATABASE_URL"
+	envPromPort    = "CL_PROMETHEUS_PORT"
+
 	envTracingEnabled         = "CL_TRACING_ENABLED"
 	envTracingCollectorTarget = "CL_TRACING_COLLECTOR_TARGET"
 	envTracingSamplingRatio   = "CL_TRACING_SAMPLING_RATIO"
 	envTracingAttribute       = "CL_TRACING_ATTRIBUTE_"
 	envTracingTLSCertPath     = "CL_TRACING_TLS_CERT_PATH"
+
+	envTelemetryEndpoint         = "CL_TELEMETRY_ENDPOINT"
```

Comment:
#nit naming unification

```suggestion
	envTelemetryCollectorTarget         = "CL_TELEMETRY_COLLECTOR_TARGET"
```

Reply:
This is also aligned with the `beholder.Config`